src/main/java/org/folio/marc4ld/service/ld2marc/processing/combine/impl/AbstractDataFieldCombiner.java

```diff
@@ -0,0 +1,90 @@
+package org.folio.marc4ld.service.ld2marc.processing.combine.impl;
+
+import static java.util.Objects.isNull;
+import static org.apache.commons.lang3.ObjectUtils.notEqual;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.marc4ld.service.ld2marc.processing.combine.DataFieldCombiner;
+import org.marc4j.marc.DataField;
+import org.marc4j.marc.Subfield;
+
+public abstract class AbstractDataFieldCombiner implements DataFieldCombiner {
+
+  private static final char EMPTY = ' ';
+
+  private DataField combinedField;
```

Comment:
Something to consider -

Both 245 and 010 are non-repeatable marc fields. Hence, there will be only one occurrence of `combinedField` for those 2 marc fields. Hence, a single occurrence of `combinedField` (as given in this class) is perfectly fine.

However, there are a lot of marc fields that can repeat (say, marc 020, 264 etc). In such cases, we will have to maintain a list of combinedFields (`List< DataField> combinedFields;`). 

It is difficult to design this class for such future requirements without knowing the actual requirements.  May be, we will rename this class to `AbstractNonRepeatableDataFieldCombiner` and a create different class for `RepeatableDataField` in future.

So, I am fine to leave this class like this for now. However, this is something we have to keep in mind.